src/modules/airspeed_selector/AirspeedValidator.hpp

```diff
@@ -181,8 +181,10 @@ class AirspeedValidator
 	bool _first_principle_check_failed{false}; ///< first principle check has detected failure
 	float _aspd_fp_t_window{0.f}; ///< time window for first principle check
 	FilteredDerivative<float> _IAS_derivative; ///< indicated airspeed derivative for first principle check
-	AlphaFilter<float> _throttle_filtered; ///< filtered throttle for first principle check
+	static constexpr float throttle_filter_tconst = 0.5f; ///< time constant for throttle filter
```

Comment:
This is the style we should strive for:
https://google.github.io/styleguide/cppguide.html#Constant_Names

Using `=` is correct, but all the variables above and below are using `{}`

And I know each line here has a comment, but it shouldn't be a rule, especially if it doesn't add any information. That's just noise. If `tconst` requires a comment, let's call it `timeConstant`